dbilo-siuntos-frontend/src/pages/HomePage.tsx

```diff
@@ -1,21 +1,52 @@
 import React from 'react';
+import Box from '@mui/material/Box';
+import Container from '@mui/material/Container';
+import Grid from '@mui/material/Grid';
+import { alpha, Typography } from '@mui/material';
 
-import { DateDisplay } from '../components/DateDisplay';
+import homepageImage from '../assets/homepage-image.jpg';
 
 export const HomePage: React.FC = () => {
   return (
-    <div
-      style={{
-        position: 'relative',
-        width: '100%',
-        display: 'flex',
-        justifyContent: 'center',
-        alignItems: 'center',
-        flexDirection: 'column',
-      }}
-    >
-      <h1 style={{ fontSize: '4em' }}>Hello world!</h1>
-      <DateDisplay />
-    </div>
+    <Container disableGutters maxWidth={false}>
+      <Box minHeight="450px" bgcolor={(theme) => alpha(theme.palette.secondary.main, 0.5)}>
+        <Container maxWidth="lg">
+          <Grid spacing={0} sx={{ marginTop: '10px' }} container>
+            <Grid item md={6} sm={12}>
+              <div
+                style={{
+                  display: 'flex',
+                  justifyContent: 'center',
+                  alignItems: 'center',
+                  minHeight: '450px',
```

Comment:
I'm afraid of these "hardcoded" value, like "450px" etc. Just asking, if they are used once that's ok, if they're connected so that one changes -> so should another then would be better to keep as consts :)

Reply:
I'm going to change how this is handled with the responsive homepage task. But yeah, in general I would not leave these values hardcoded like this